Bukkit/src/main/java/com/plotsquared/bukkit/listener/BlockEventListener.java

```diff
@@ -121,6 +121,23 @@ public class BlockEventListener implements Listener {
             Material.TURTLE_EGG,
             Material.TURTLE_SPAWN_EGG
     );
+    private static final Set<Material> SNOW;  // needed as Tag.SNOW isn't present in 1.16.5
+
+    static {
+        if (PlotSquared.platform().serverVersion()[1] < 17) {
+            SNOW = Set.of(
+                    Material.SNOW,
+                    Material.SNOW_BLOCK
+            );
+        } else {
+            SNOW = Set.of(
+                    Material.SNOW,
+                    Material.SNOW_BLOCK,
+                    Material.POWDER_SNOW // only since 1.17
+            );
+        }
+    }
+
```

Comment:
I personally would be a bigger fan of something like 
```
private static final Set<Material> SNOW_BLOCKS = Stream.of(Material.values())
            .filter(material -> material.name().contains("SNOW"))
            .filter(Material::isBlock)
            .collect(Collectors.toSet());
```
Maintainability would be better in that case, as static collections like that might get lost and forgot on updates. The only problems I've found on my testing is, that the cauldron filled with powedered snow ([SNOW_BLOCK, **POWDER_SNOW_CAULDRON**, POWDER_SNOW, SNOW]) is present in that set as well. But I don't think that's that big of an issue, as that block wouldn't call the BlockFormEvent.

Reply:
> Maintainability would be better in that case, as static collections like that might get lost and forgot on updates.
I like that it would be more maintainable but I am not a big fan of it iterating over all the materials just for snow blocks.
That being said, it is the most maintainable solution and readable so I am fine with either Pierre or SirYwell's solution.

Reply:
@PierreSchwang is this blocking feedback? As indicated by your approval, I'd say the opposite.

Reply:
Nope, as I said this would just be an opinion how it could potentially be more future proof. But I approved nevertheless, as this can be merged either way. Would've appreciate an opinion by Kalle tho